test/garden/CMakeLists.txt

```diff
@@ -6,12 +6,10 @@ add_executable( ${REGRESS_NAME} ${SFCGAL_REGRESS_GARDEN_TEST_SOURCES} )
 
 target_link_libraries( ${REGRESS_NAME}
 	SFCGAL
-	${Boost_LIBRARIES}
-	${CGAL_LIBRARIES}
+        CGAL::CGAL
+        CGAL::CGAL_Core
```

Comment:
@sloriot I don't know this syntax. What differences with the previous situation ? (i.e. what do you fix here ?)

Reply:
That syntax is a new one, using imported targets. That syntax exists since CGAL-4.7, but it has been powered-up with CGAL-4.12-beta1, because now the imported targets know about their own dependencies: if you link with `CGAL::CGAL`, then you get automatically:
  - the include path of CGAL,
  - the include path of Boost, GMP, MPFR...,
  - link with CGAL,
  - link with with Boost Thread (if needed), and with GMP, and MPFR.